src/test/unit/highlights/highlight_grid.test.ts

```diff
@@ -630,6 +630,26 @@ describe("processHighlightCellsEvent", () => {
                 },
             ],
         },
```

Comment:
Thanks for carrying this over. If you would be so kind, would you mind adding a case for where the `Math.max` will trigger return 0, when calculating `eolCol`?